app/models/managed_interactive.rb

```diff
@@ -137,13 +137,22 @@ def to_authoring_hash
     hash = to_hash
     hash[:id] = id
     hash[:linked_interactive_id] = linked_interactive_id
+    hash[:linked_interactive_type] = linked_interactive_type
     hash[:aspect_ratio] = aspect_ratio
+    hash[:page_item_id] = page_item ? page_item.id : nil
+    hash
+  end
+
+  def to_authoring_preview_hash
```

Comment:
I can see where this would be confusing so I've removed it in favor of just the runtime hash in https://github.com/concord-consortium/lara/pull/601/commits/fe1dc037b40bd1416e3d491b655a486550d957b8

Originally I added this so I could see the interactive links as they were set but since that isn't part of the runtime init it could possibly cause issues if devs only used the preview to test the runtime,

Reply:
Update - I remember know why I did it.  We need to add items to the runtime hash that are otherwise set as data items during runtime.  I'm going to backout the commit I just made and instead flesh out the `authoring_preview_hash` so that it can generate the sameish (minus the run stuff) `interactiveInit` message.

Reply:
This is now fixed in https://github.com/concord-consortium/lara/commit/5e3d94ddf04451a5ec9c4fb9dc97dab31efe6a25 which implements the full runtime initInteractive message in authoring previews.